pkg/plugin/v2/scaffolds/internal/templates/controller/controller_suitetest.go

```diff
@@ -31,6 +31,11 @@ type SuiteTest struct {
 	file.TemplateMixin
 	file.RepositoryMixin
 	file.MultiGroupMixin
+
+	// WireResource defines the api resources are generated or not.
+	// Ordered the struct this way to save the memory by organizing struct
```

Comment:
Same as in the controller, code clarity way more important than memory efficiency here.